### PR TITLE
TIP-1129: Clean TIP-1129

### PR DESCRIPTION
**Pull request by pierallard**
Burn

### CODE REVIEW

File: src/Akeneo/Pim/Enrichment/Component/Product/Normalizer/Indexing/ProductAndProductModel/ProductModelNormalizer.php [hunk 1]
```diff
@@ -4,115 +4,7 @@
...
-        $attributes = $this->attributesProvider->getAttributes($entityWithFamilyVariant);
-        $attributeCodes = array_map(function (AttributeInterface $attribute) {
-            return $attribute->getCode();
-        }, $attributes);
-
-        sort($attributeCodes);
-
-        return $attributeCodes;
-    }
+    const INDEXING_FORMAT_PRODUCT_AND_MODEL_INDEX = 'indexing_product_and_product_model';
```
Comment (ahocquard) [inline src/Akeneo/Pim/Enrichment/Component/Product/Normalizer/Indexing/ProductAndProductModel/ProductModelNormalizer.php]: no way to remove it somewhere else? (ValueCollectionNormalizer for example)

File: src/Akeneo/Pim/Enrichment/Bundle/Doctrine/ORM/Query/GetElasticsearchProductModelProjection.php [hunk 2]
```diff
@@ -20,14 +19,8 @@
  */
 class GetElasticsearchProductModelProjection implements GetElasticsearchProductModelProjectionInterface
```
Comment (ahocquard) [inline src/Akeneo/Pim/Enrichment/Bundle/Doctrine/ORM/Query/GetElasticsearchProductModelProjection.php]: it's in `Akeneo\Pim\Enrichment\Bundle\Doctrine\ORM\Query` but it should be in `Storage/Sql/ElasticsearchProjection/GetElasticsearchProductModelProjection`

File: src/Akeneo/Pim/Enrichment/Bundle/Doctrine/ORM/Query/GetElasticsearchProductModelProjection.php [hunk 3]
```diff
@@ -20,14 +19,8 @@
...
 {
-    /** @var GetValuesAndPropertiesFromProductModelCodes */
```
Comment (nmarniesse) [inline src/Akeneo/Pim/Enrichment/Bundle/Doctrine/ORM/Query/GetElasticsearchProductModelProjection.php]: Think this class does not belong to namespace `ORM` anymore. Maybe in `Storage\Query` or `Product\Query`.  
If you prefer we add a card where we re-organize class like that

File: src/Akeneo/Pim/Enrichment/Component/Product/Normalizer/Indexing/ProductAndProductModel/ProductModelNormalizer.php (hunk 1 above)
Comment (nmarniesse) [inline src/Akeneo/Pim/Enrichment/Component/Product/Normalizer/Indexing/ProductAndProductModel/ProductModelNormalizer.php]: Is it possible to move this constant in another class? In order to remove this class.

File: src/Akeneo/Pim/Enrichment/Bundle/Resources/config/serializers_indexing.yml [hunk 4]
```diff
@@ -116,23 +114,6 @@ services:
         tags:
             - { name: pim_indexing_serializer.normalizer, priority: 90 }
 
-    pim_catalog.normalizer.indexing_product_and_product_model.product_model:
```
Comment (mmetayer) [inline src/Akeneo/Pim/Enrichment/Bundle/Resources/config/serializers_indexing.yml]: Add these in the CHANGELOG please

**Pull request merged**